af/fastpath/fastpath/core.py

```diff
@@ -696,6 +696,29 @@ def score_http_invalid_request_line(msm):
     return scores
 
 
+def get_http_header(resp, header_name, case_sensitive=False):
```

Comment:
I _assume_ you are adding this `case_sensitive` switch because you anticipate it would be useful in the future to have this functionality in this function to perform comparison for tests that change the header case?

Reply:
Yeah, the idea behind this is that there might be cases in which we actually care to search for a header with case_sensitivity turned on (I am thinking for example of the middlebox tests where some header mangling may be performed by the middlebox).
Since it seemed easy to support this use case, I figured I would add a flag for it from the get-go.

Alternatively, if you feel that having unused functionality is not idea, we can also drop this and add it only when we actually need it.

Reply:
> [...] the idea behind this is that there might be cases in which we actually care to search for a header with case_sensitivity turned on (I am thinking for example of the middlebox tests where some header mangling may be performed by the middlebox)

A-ha! This totally make sense, but it confused me because I did not think about it. Perhaps add some documentation to the function explaining why we have this support?

Thanks in general for adding support for the headers list, it makes me happy to see it!